mongodb_base_block.py

```diff
@@ -84,6 +89,9 @@ class MongoDBBase(Block):
     database = StringProperty(title='Database Name', default="test")
     collection = ExpressionProperty(title='Collection Name', default="signals")
     creds = ObjectProperty(Credentials, title='Credentials')
+    mongo_version = SelectProperty(MongoVersion,
+                                   default=MongoVersion.v2,
+                                   title="MongoDB Version")
```

Comment:
This is a great idea, thanks. Can't think of a better way to do a version compare, this seems better than what I would have done with `pymongo.version[0] == '3'` haha, so I'll go with this.